src/pages/en/reference/api-reference.mdx

```diff
@@ -553,7 +569,7 @@ export function get({ redirect }: APIContext) {
 }
 ```
 
-See also: [Astro.redirect](/en/guides/server-side-rendering/#astroredirect)
+See also: [Astro.redirect()](#astroredirect)
```

Comment:
Missed this previously but this should probably be in backticks? This page is a bit of a mixed bag doing these links with and without in different places but I’d go with standardising to with code backticks.

```suggestion
See also: [`Astro.redirect()`](#astroredirect)
```

That is all. Merge often, merge early!